Finding Pairs.cpp

```diff
@@ -0,0 +1,50 @@
+/*Given an array A of N numbers, find the number of distinct pairs (i, j) such that j >=i and A[i] = A[j].
+First line of the input contains number of test cases T. Each test case has two lines, first line is the number N, followed by a line consisting of N integers which are the elements of array A.
+For each test case print the number of distinct pairs.
+
+Constraints
```

Comment:
I am merging your code, but this is a code-template repository, not problem solving.